creators/templatetags/creator_books.py

```diff
@@ -31,12 +31,17 @@ def get_creator_books(creator: Creator, perms):
     while series_len < len(series):
         series_len = len(series)
         series = series | set(Series.objects.filter(part_of_series__in=series))
+    subworks = set(Work.objects.filter(creatortowork__creator=creator))
```

Comment:
Can't this be done in-line in the query below? Fetching everything and then doing a `where in` query is not the most efficient way possible.